cmd/key_generate.go

```diff
@@ -193,18 +193,8 @@ func (c *KeyGenerateCommand) Run(args []string) int {
 		}
 	}
 
-	knownHosts := keyscanHosts(strings.Split(c.knownHosts, ","))
-
-	if len(knownHosts) == 0 {
+	if err := setSshKnownHostsSecret(sshKnownHostsSecret, c.knownHosts, c.repo); err != nil {
 		c.UI.Error("Error: could not set SSH known hosts.")
```

Comment:
Needs to be added back after this line

```go
c.UI.Error(err.Error())
```

Reply:
Thanks; take a look now @swalkinshaw

Reply:
Formatting is off 😅  You can run `gofmt -s -w .`

Reply:
Ah, sorry about that; the past few PRs have been my first time writing in Go so thanks for letting me know if the command 👍